docs/RELEASE.md

```diff
@@ -0,0 +1,79 @@
+# AWS IAM Authenticator Release Process
+
+NOTE: your github user must have the required permissions and you must have generated a github token for that user.
+
+## Choosing the release version and branch
+
+Using semantic versioning, pick a release number that makes sense by bumping the major, minor or patch release version.  If its a major or minor release (backwards incompatible changes, and new features, respectively) then you will want to start this process with an alpha release first.  Here are some examples:
+
+Bumping a minor version after releasing a new feature:
+```
+v0.4.5 -> v0.5.0-alpha.0
```

Comment:
Can we make these examples 1.x.y? Technically v0.x.y are unstable and can have breaking changes at any time.

Separately, its probably worth cutting a v1.0.0 since we're pretty stable

Reply:
That's fair on both points.  We can probably work towards a 1.0 but should have a bit of a triage and bug bash before cutting it.